cli/tests/unit/kv_queue_test.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { assertEquals } from "./test_util.ts";
+
+Deno.test({}, async function queueTestDbClose() {
+  const db: Deno.Kv = await Deno.openKv(":memory:");
+  db.close();
+  try {
+    await db.listenQueue(() => {});
+  } catch (e) {
+    assertEquals(e.message, "already closed");
+  }
```

Comment:
What if it doesn't throw? Right now the test will pass if it doesn't throw.

Reply:
Nice catch :).
Added `assertFalse(false)` after line 9 so if doesn't throw it will fail.